src/pages/home/report/ReportActionCompose.js

```diff
@@ -542,31 +722,43 @@ class ReportActionCompose extends React.Component {
                 this.props.isComposerFullSize && styles.chatItemFullComposeRow,
             ]}
             >
-                {shouldShowReportRecipientLocalTime
-                    && <ParticipantLocalTime participant={reportRecipient} />}
-                <View style={[
-                    (!isBlockedFromConcierge && !this.props.disabled && (this.state.isFocused || this.state.isDraggingOver))
-                        ? styles.chatItemComposeBoxFocusedColor
-                        : styles.chatItemComposeBoxColor,
-                    styles.flexRow,
-                    styles.chatItemComposeBox,
-                    this.props.isComposerFullSize && styles.chatItemFullComposeBox,
-                    hasExceededMaxCommentLength && styles.borderColorDanger,
-                ]}
+
+                {shouldShowReportRecipientLocalTime && (
+                <ParticipantLocalTime participant={reportRecipient} />
+                )}
+                <View
+                    style={[
+                        !isBlockedFromConcierge
+                && !this.props.disabled
+                && (this.state.isFocused || this.state.isDraggingOver)
+                            ? styles.chatItemComposeBoxFocusedColor
+                            : styles.chatItemComposeBoxColor,
+                        styles.flexRow,
+                        styles.chatItemComposeBox,
+                        this.props.isComposerFullSize && styles.chatItemFullComposeBox,
+                        hasExceededMaxCommentLength && styles.borderColorDanger,
+                    ]}
                 >
                     <AttachmentModal
-                        headerTitle={this.props.translate('reportActionCompose.sendAttachment')}
+                        headerTitle={this.props.translate(
```

Comment:
Please undo this change. Generally we prefer PRs to be more focused and include only minimal changes. This makes them easier to review by keeping the diff smaller.